app/src/main/java/com/concordium/wallet/ui/transaction/transactiondetails/TransactionDetailsActivity.kt

```diff
@@ -249,13 +246,13 @@ class TransactionDetailsActivity : BaseActivity() {
             binding.blockHashLayout.visibility = View.VISIBLE
             binding.blockHashLayout.setValue(
                 getString(R.string.transaction_details_block_hash_submitted),
-                true
+                false
             )
         } else if (ta.transactionStatus == TransactionStatus.ABSENT) {
             binding.blockHashLayout.visibility = View.VISIBLE
             binding.blockHashLayout.setValue(
                 getString(R.string.transaction_details_block_hash_failed),
-                true
+                false
```

Comment:
If we are displaying the entire address, then I think we should also display the entire block hash / transaction hash. (But I'm not sure why we are not already doing this. :thinking: )
![Screenshot from 2023-07-03 10-32-54](https://github.com/Concordium/concordium-reference-wallet-android/assets/87635671/a5a6b7e7-5cfc-43c3-9c34-9696aadbfdf3)